services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpec.groovy

```diff
@@ -175,6 +179,49 @@ class SwitchValidationSpec extends BaseSpecification {
             switchHelper.verifyRuleSectionsAreEmpty(it, ["missing", "excess"])
         }
 
+        and: "Flow validation shows discrepancies"
+        def flowValidateResponse = northbound.validateFlow(flow.id)
+        flowValidateResponse.size() == 2
+        flowValidateResponse[0].each { direction ->
+            assert direction.discrepancies.size() == 2
+
+            def rate = direction.discrepancies[0]
+            assert rate.field == "meterRate"
+            assert rate.expectedValue == newBandwidth.toString()
+            assert rate.actualValue == flow.maximumBandwidth.toString()
+
+            def burst = direction.discrepancies[1]
+            assert burst.field == "meterBurstSize"
+            Long newBurstSize = switchHelper.getExpectedBurst(srcSwitch.dpId, newBandwidth)
+            verifyBurstSizeIsCorrect(newBurstSize, burst.expectedValue.toLong())
+            verifyBurstSizeIsCorrect(srcSwitchBurstSize, burst.actualValue.toLong())
+
+            direction.flowRulesTotal == 2
+            direction.switchRulesTotal == srcSwitchAmountOfRules + 2
```

Comment:
I don't understand why `direction.switchRulesTotal` would show the amount of rules on src switch. Why not total sum of rules on all the flow's switches? This looks incorrect for me

Reply:
you are right
develop branch was fixed and test was fixed too